cilium-cli/multicast/multicast.go

```diff
@@ -501,7 +501,7 @@ func (m *Multicast) AddAllNodes() error {
 				}
 			}
 			if cnt == 0 {
-				fmt.Fprintf(m.params.Writer, "Unable to add any node to multicast group %s in %s\n", m.params.MulticastGroupIP, pod.Spec.NodeName)
+				fmt.Fprintf(m.params.Writer, "No available nodes to be added to multicast group %s in %s\n", m.params.MulticastGroupIP, pod.Spec.NodeName)
```

Comment:
this is kinda misleading message for user, precisely there has been no available nodes to be added. (it did not fail to add.)

```console
$ cilium multicast add --group-ip 239.255.0.1
Unable to add any node to multicast group 239.255.0.1 in tomoyafujita
$ cilium multicast list group
Node           Group         
tomoyafujita   239.255.0.1   
$ cilium multicast list subscriber --all
Node           Group         Subscriber   Type
tomoyafujita   239.255.0.1                

```